manifests/0000_31_cluster-baremetal-operator_06_deployment.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
I suspect "make manifests" is going to overwite this. maybe add it here https://github.com/openshift/cluster-baremetal-operator/blob/master/config/profiles/default/kustomization.yaml#L2

Reply:
done, thanks!